styles/_task-board.scss

```
@@ -75,3 +86,117 @@
     }
   }
 }
+
+.watch-details {
+  border-radius: 15px;
+  display: flex;
+  justify-content: space-evenly;
+  height: 400px;
+
+  .poster-container {
+    width: 25%;
+    display: flex;
+    align-items: center;
+  }
+  img {
+    max-width: 100%;
+    margin: auto;
+  }
+  .info {
+    width: 45%;
+    color: $font-color;
+    .info-header {
+      display: flex;
+      flex-direction: column;
+      align-items: flex-start;
+      padding: 0.6em;
+      border-bottom: 1.5px solid rgba($color: $font-color, $alpha: .5);
+
+      .title {
+        font-size: 2em;
+        font-weight: 400;
+      }
+
+      .genre {
+        background-color: $font-color;
+        color: $theme-color;
+        padding: 0.1em 0.3em;
+        border-radius: 5px;
+      }
+    }
+
+    .description {
+      border-bottom: 1.5px solid rgba($color: $font-color, $alpha: .5);
+    }
+
+    .director {
+      display: flex;
+      .director-title {
+        font-weight: 900;
+        margin-right: 1em;
+      }
+      border-bottom: 1.5px solid rgba($color: $font-color, $alpha: .5);
+    }
+
+    .cast {
+      .cast-title {
+        font-weight: 900;
+      }
+    }
+  }
+  .more {
+    width: 25%;
+    .ratings {
+      height: 65%;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-evenly;
+      border-bottom: 1.5px solid rgba($color: $font-color, $alpha: .5);
+      span {
+        font-weight: 700;
+        font-size: 1.6em;
+        position: absolute;
+        left: 65%;
+        top: 30%;
+      }
+      .IMDB-rating {
+        position: relative;
+        img {
+          width: 40%;
+          margin-left: 15%;
+        }
+        span {
+          top: 20%;
+        }
+      }
+      .RT-rating {
+        position: relative;
+        img {
+          width: 30%;
+          margin-left: 20%;
+        }
+      }
+      .MC-rating {
+        position: relative;
+        img {
+          width: 30%;
+          margin-left: 20%;
+        }
+      }
+    }
+
+    .streaming-guide {
+      .streaming {
+        font-size: 1.2em;
+        font-weight: 700;
+      }
+
+      img {
+        width: 85%;
+        margin-top: 7%
+      }
```

I'd change to max-width here as well on the `img`s. I think they look a little better being a bit smaller, but I'll let you judge that.

Also made `.streaming-guide` display-flex and some other stuff to center that content.

```suggestion
      .IMDB-rating {
        position: relative;
        img {
          max-width: 30%;
          margin-left: 15%;
        }
        span {
          top: 20%;
        }
      }
      .RT-rating {
        position: relative;
        img {
          max-width: 20%;
          margin-left: 20%;
        }
      }
      .MC-rating {
        position: relative;
        img {
          max-width: 20%;
          margin-left: 20%;
        }
      }
    }

    .streaming-guide {
      display: flex;
      flex-direction: column;
      justify-content: center;
      align-items: center;
      padding: 0;

      .streaming {
        font-size: 1.2em;
        font-weight: 700;
      }

      a {
        display: flex;
        justify-content: center;
        img {
          align-self: center;
          max-width: 75%;
          margin-top: 7%;
        }
      }
    }
  }
}
```